File: alignment/src/test/scala/com/vitreoussoftware/bioinformatics/alignment/AlignmentTest.scala
package com.vitreoussoftware.bioinformatics.alignment

import com.vitreoussoftware.test.BehaviorSpec
import htsjdk.samtools.SAMRecord
import org.scalatest.Outcome

/**
 * Created by John on 10/5/14.
 */
public abstract class AlignmentTest extends BehaviorSpec {
  describe("Alignment") {
    it("can be converted to a SAMRecord") {
      () => {
        0 should be(0);
      }
    }
  }
}

File: alignment/src/test/scala/com/vitreoussoftware/bioinformatics/alignment/AlignmentTest.java
package com.vitreoussoftware.bioinformatics.alignment;

import nl.jqno.equalsverifier.EqualsVerifier;
import org.junit.Test;

/**
 * Created by John on 12/4/2016.
 */
public class AlignmentTest {
    @Test
    public void equalsContract() {
        EqualsVerifier.forClass(Alignment.class)
                .verify();
    }
}